let a = 10;

try {
    if (a==10) {
        console.log(true);
    }
} catch (error) {
    console.log(error.message);
} 